Code/Framework/AzCore/Platform/Android/platform_android_files.cmake

```diff
@@ -86,7 +86,7 @@ set(FILES
 )
 if (LY_TEST_PROJECT)
     ly_add_source_properties(
-        SOURCES AzCore/Android/Utils.cpp
+        SOURCES Platform/Android/AzCore/Android/Utils.cpp
```

Comment:
Actually this should be relative to the current directory.
If anything what should be used here is the `${CMAKE_CURRENT_LIST_DIR}` to get the current directory of this cmake file

Reply:
Like this?

````
SOURCES ${CMAKE_CURRENT_LIST_DIR}/AzCore/Android/Utils.cpp
````

Reply:
Yes, that should work

Reply:
Cool, thank. I'll test it locally and update the PR if it passes.

Reply:
It worked, updating PR.